app/components/views/DexPage/CreateWalletsPage/CreateWalletsPage.jsx

```diff
@@ -47,6 +48,7 @@ const CreateWalletsPage = () => {
   return (
     <div className="flex-column align-start">
       {!dexBTCWalletRunning ? (
+        /* !dexBTCWalletConfigured ? ( offer choice between native bitcoind RPC and native SPV ) */
```

Comment:
Depending on their choice in this hypothetical dialog, the Decrediton would call `btcCreateWalletDex` with a `walletType` arg of either `"bitcoindRPC"` or `"SPV"`.